### PR TITLE
Add shortcut for the Multiplayer window

### PR DESCRIPTION
**Pull request by Nubbie**
Adds a shortcut for the player list window, good for server owners etc who wants to get it up quickly

~~Known problem:
The menu is able to be opened in singleplayer
Can't fix that and if someone wants to, you're more than welcome to take this code~~

### CODE REVIEW

**Comment by Nubbie** [comment]
pic for listr

**Comment by IntelOrca** [comment]
> The menu is able to be opened in singleplayer

Its quite simple, use ```network_get_mode``` to test whether the game is in multiplayer or not. If it isn't don't open the window.

**Comment by Nubbie** [comment]
Too bad Im not a good programmer tho :P 
Still the menu maybe is usable in singleplayer too if you want to edit groups

**Comment by IntelOrca** [comment]
> Too bad Im not a good programmer tho :P 

You won't get better unless you try.

**Comment by Nubbie** [comment]
I sure do will try if we're going to limit it or not, dunno if you want to restrict it just to multiplayer because, like I said, what about the groups? Maybe it's something someone want to do in peace in singleplayer xox

**Comment by IntelOrca** [comment]
Restrict it to multiplayer, it can be changed later if necessary.

**Comment by Nubbie** [comment]
Corrected the problem with it being able to open both in SP and & MP, only opens in MP now, thanks IntelOrca :+1:

File: src/interface/keyboard_shortcut.c [hunk 1]
```diff
@@ -537,6 +537,12 @@ static void shortcut_windowed_mode_toggle()
 	platform_toggle_windowed_mode();
 }
 
+static void shortcut_show_multiplayer()
+{
+	if (network_get_mode() != NETWORK_MODE_NONE)
+	window_multiplayer_open();
```
Comment (IntelOrca) [inline src/interface/keyboard_shortcut.c]: Please indent this, all code in control blocks should be indented.

**Comment by Nubbie** [comment]
**Ready to merge**

**Pull request merged**